pilot/cmd/pilot-agent/main.go

```diff
@@ -298,24 +296,6 @@ var (
 				defer stsServer.Stop()
 			}
 
-			// Start a local DNS server on 15053, forwarding to DNS-over-TLS server
```

Comment:
Maybe leave it in until the replacement is ready ? Not a big deal if we expect envoy dns-tls support to be ready soon. This exist primarily to provide a secure option.